### PR TITLE
Add plastic

### PR DESCRIPTION
**Pull request by Amjad50**
The TUI itself is in `plastic_tui` crate, check the demo as well :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +125,7 @@ Aside from those listed here, many other apps and libraries can be easily be fou
 - [Thardians](https://gitlab.com/thustle/thardians-rs) - Space Invaders for the terminal.
 - [tic-tac-toe](https://github.com/thomas-mauran/tic-tac-toe) - Terminal-based tic tac toe game.
 - [wordl](https://github.com/palerdot/wordl-rs) - Terminal-based Wordle game. Web like experience with keyboard hints and guess reveal animations.
+- [plastic](https://github.com/Amjad50/plastic) - NES emulator with extra ui implemented in ratatui.
```
Comment (orhun) [inline README.md:128]: Can you place this accordingly to the alphabetical order? :)

**Review by orhun — APPROVED** [review]
This is super cool!

File: README.md (hunk 1 above)
Comment (Amjad50) [reply README.md:128]: ah didn't notice, yes, I"ll fix it

Comment (Amjad50) [reply README.md:128]: Fixed

Comment (Amjad50) [reply README.md]: would be good to maybe have this as part of the linter CI

**Pull request merged**